pvlib/spectrum/mismatch.py

```diff
@@ -531,26 +531,69 @@ def spectral_factor_firstsolar(precipitable_water, airmass_absolute,
 
 def spectral_factor_sapm(airmass_absolute, module):
     """
-    Calculates the SAPM spectral loss coefficient, F1.
+    Calculates a module-specific SAPM spectral loss coefficient, :math:`f_1`,
+    equivalent to the spectral mismatch factor, using absolute
+    (pressure-corrected) air mass, :math:`AM_a`.
```

Comment:
```suggestion
    Calculates the spectral mismatch factor, :math:`f_1`,
    using the Sandia Array Performance Model approach.
```

Unless there is a reason to call the output "spectral loss coefficient". I don't know of a reason.

Reply:
I don't care for "spectral loss" since it may well be a gain (f1 > 1). I would change the Returns text to match the introduction sentence.